src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/person/UnmergeService.kt

```diff
@@ -36,7 +43,17 @@ class UnmergeService(
   private suspend fun processUnmergingOfRecords(unmergeEvent: UnmergeEvent, reactivatedPersonCallback: () -> PersonEntity?, unmergedPersonCallback: () -> PersonEntity?) {
     val unmergedPersonEntity = retrieveUnmergedPerson(unmergeEvent, unmergedPersonCallback)
     val reactivatedPersonEntity = retrieveReactivatedPerson(unmergeEvent, reactivatedPersonCallback)
+
+    val beforeDataDTO = Person.convertEntityToPerson(unmergedPersonEntity)
+    val beforeData = objectMapper.writeValueAsString(beforeDataDTO)
+
     unmergeRecords(unmergeEvent, reactivatedPersonEntity, unmergedPersonEntity)
+
+    val sourceSystemId = extractSourceSystemId(reactivatedPersonEntity)
+    val processedDataDTO = Person.convertEntityToPerson(reactivatedPersonEntity)
+    val processedData = objectMapper.writeValueAsString(processedDataDTO)
```

Comment:
These lines or very similar ones appear just before all the calls to the `eventLoggingService`
Could we perhaps pass in two `PersonEntities` and do all this work in the `eventLoggingService`?
It would make it easier to add in the logging for the next thing we decide to log